From:  "Thomas Weber" <tweber@debian.org>
Subject: Don't crash on calling addpath('./')
Description: Octave 3.2 crashes when calling something like 
 echo "addpath('./'); addpath('./')" | octave
 This patch originates from a larger patch from upstreams's development version.
Origin: partly upstream, http://hg.savannah.gnu.org/hgweb/octave/rev/4acae5e46738
Bug-Debian: http://bugs.debian.org/592517
Applied-Upstream: http://hg.savannah.gnu.org/hgweb/octave/rev/4acae5e46738

--- a/src/load-path.cc
+++ b/src/load-path.cc
@@ -653,8 +653,6 @@
 
   if (i != dir_info_list.end ())
     move (i, false);
-  else
-    panic_impossible ();
 }
 
 void
